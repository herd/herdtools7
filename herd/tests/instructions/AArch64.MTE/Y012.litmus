AArch64 Y012
(* IRG with SP *)
{
  int x=100;
  0:SP=x;
  0:GCR_EL1=0x17FFF;  (* RRND=1, Exclude=0x7FFF *)
}

P0        ;
IRG SP,SP ;

forall (0:SP=x:t15)

