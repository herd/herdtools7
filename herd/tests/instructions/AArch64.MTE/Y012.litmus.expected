Test Y012 Required
States 1
0:SP=x:t15;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:SP=x:t15)
Observation Y012 Always 1 0
Hash=65b2631776ae487f3d1cff92c9dd066f

